CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## securesystemslib v0.19.0
+
+### Added
+* Enable setting which GPG client to use through an environment variable (#315)
+
+### Changed
+* Dropped support for EOL Python 3.5 and add support for Python 3.9 (#314)
+* Converted the default local storage backend, FilesystemBackend, to be a
+  singleton (#302)
+* Migrated CI from travis-ci.org to travis-ci.com (#303) then later to GitHub
+  Actions (#324)
+* Bump dependencies: cffi (#306, 329), cryptography (#322, #333)
```

Comment:
```suggestion
* Bump dependencies: cffi (#306, 329), cryptography (#322, #333). Latest version of cryptography is no longer used on Python 2 as that is not supported
```

Reply:
This is a nice addition, makes sense to be explicit. Thanks.